### ISSUE TITLE
{Freeway-Bug}[Adaptive Card> ActivityUpdateWithLabel.json> Comment]: Voice access users are unable to edit text format by using "Bold[text]", "Italicize [Text]" commands under comment edit field.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#7519]
### Target Platforms

UWP

### SDK Version

3.0.2206.2001

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS Version: 22H2 (Build 25126.1000)
Application: Adaptive Cards UWP
Application Version: 3.0.2206.2001
Tool: Voice access

**Repro Steps:**

1. Turn on the Voice Access.
2. Open Adaptive Cards UWP application.
3. Command: 'Voice Access wake up'
4. Command: 'Show numbers'
5. Give the number which is defined for 'ActivityUpdateWithLabel.json'
6. Command: 'Move to 'Comment'
7. Command: 'click'
8. Dictation mode: Type any text.
9. Command mode: "Bold[text]", "Italicize [Text]"
 
**Actual Result:**
Voice access users are unable to edit text format by using "Bold[text]", "Italicize [Text]" commands under comment edit field. On giving command 'Italicize"Text"' particular text get invisible on the comment edit field

**Expected Result:**
Voice access users should be able to edit text format by using "Bold[text]", "Italicize [Text]" commands under comment edit field and text be properly visible to the user with action performed.

**Notes:**
This issue repro throughout for all the edit fields.

**User Impact:**
People with limited mobility (cerebral palsy, quadriplegia) who use Voice Access will find difficulty in editing the text format properly and will not be able to perform the function efficiently.
 
**MAS reference link:**
[[No Disruption of Accessibility Features - Liquid (microsoft.com)]](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.02.01#Zguide)

**Have feedback to share on Bugs?** Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

https://user-images.githubusercontent.com/104483005/174583198-22c0712f-0c3c-4398-a334-5ed5782b42bb.mp4

### Card JSON

```JSON
NA
```

**Comment by Rishikesh6521** [comment]
**Invalid Bug**

As there is not formatting toolbar options to bold or italize the text. Hence closing this bug.
